src/actions/index.js

```diff
@@ -0,0 +1,14 @@
+export const createBook = (book) => ({
+  type: 'CREATE_BOOK',
+  book
+})
+
+export const removeBook = (book) => ({
```

Comment:
We should just be passing `id` here as we don't need the whole `book` object when we are deleting.